src/benefit.contants.ts

```diff
@@ -0,0 +1,3 @@
+export const BENEFIT_CONSTANTS = {
```

Comment:
can we move this inside benefits folder?

Reply:
Ok I can move it

Reply:
pls do so, so we can merge